README.md

```diff
@@ -1,4 +1,36 @@
 # CSharp-From-Zero-To-Hero
-[![Build Status](https://dev.azure.com/almantusk/From-Zero-To-Hero/_apis/build/status/Almantask.CSharp-From-Zero-To-Hero?branchName=master)](https://dev.azure.com/almantusk/From-Zero-To-Hero/_build/latest?definitionId=5&branchName=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a358adf21c2442cd91a4827a50213ff1)](https://www.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Almantask/CSharp-From-Zero-To-Hero&amp;utm_campaign=Badge_Grade)  
-C# boot camp
+
+# Homework 1And2
+
+## Hints
+* The tasks require you to have already created an account on GitHub. Do that first!
+* If you do not understand something or need help:
+  * Ask questions in the channel bc-discussion in Discord.
+  * Check the wiki of this repository (work in progress!).
+  * Check the bc-materials channel for reference material.
+  * Check the bc-announcements channel for related stuff.
+
+## Task
+* Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2
+  * Open the link.
+  * Click the fork button in the upper right.
+  * Click the green Clone or download button and clone your repository the way that suits you best.
+  * Open your repository in Visual Studio 2019.
+  * Select the right branch in the bottom right! That name should be Chapter1/Homework/1And2 instead of master in the end.
+  * Make sure that you open Bootcamp.sln. This is called a solution. Remember this!
+* Source code from lesson 1 homework should end up on your forked branch.
+  * Open the Src folder.
+  * Open the BootCamp.Chapter project file. Remember that this is a project file!
+  * Open the Program.cs file.
+  * Put your code from homework 1 here.
+* Create a pull request to request your forked branch to be merged to the source branch.
+  * Click the pencil symbol in the bottom right and type in a commit message.
+  * Press commit button.
+  * Click the single arrow button (next to the pencil) in the bottom right.
+  * Push your commit.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4895536)